.github/workflows/build.yaml

```diff
@@ -0,0 +1,64 @@
+name: Build and Deploy
+
+on:
+  push:
```

Comment:
Won't we need to include `pull_request `here so this build will also run when a PR is raised?